app/v1/models/audit/AuditEvent.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.audit
+
+case class AuditEvent[T](auditType: String, transactionName: String, detail: T)
```

Comment:
Where are the other fields? (auditSource, eventId, tags, generatedAt)

Reply:
These are created at the creation of ExtendedDataEvent inside auditEvent method of AuditService

Reply:
Ok great